roles/display_deployment_plan/templates/plan.j2

```diff
@@ -7,6 +7,8 @@ An OpenShift cluster is about to be deployed. Please double check the provided i
 
   {{ row_format_str | format('OpenShift version', inventory.openshift_full_version) }}
 
+  {{ row_format_str | format('Kube admin vault password file', inventory.kubeadmin_vault_password_file_path | default("UNDEFINED (This means the kubeadmin credentals will be stored in plain text)")) }}
```

Comment:
This won't work, unfortunately. You'll have to add  `kubeadmin_vault_password_file_path` to the deployment plan vars manually. The reason why you can't reference vars from the inventory directly is that Ansible will throw a templating error if an invalid inventory file is provided. Additionally, having it under the `inventory` key helps avoiding infinite loops caused by name conflicts.

You could start by extending the `messages` dict with your custom error message "UNDEFINED (This means...)"  here: https://github.com/redhat-partner-solutions/crucible/blob/main/roles/display_deployment_plan/tasks/main.yml#L27-L30

The second step is to add `kubeadmin_vault_password_file_path` to the `inventory` dict, referencing the new default error message. This can be added here: https://github.com/redhat-partner-solutions/crucible/blob/main/roles/display_deployment_plan/tasks/main.yml#L34-L48
```yaml
inventory:
    kubeadmin_vault_password_file_path: "{{ kubeadmin_vault_password_file_path | default(messages.kubeadmin_vault_file_path_missing) }}"
```

Then you should be able to print the row without the `default`:
```yaml
{{ row_format_str | format('Kube admin vault password file', inventory.kubeadmin_vault_password_file_path) }}
```

Reply:
Ah yes sorry that was me not reading the task properly.